README.rst

```diff
@@ -6,6 +6,7 @@ CrateDB Cloud Documentation
 
 Documentation for `CrateDB Cloud`_.
 
+
```

Comment:
Do we start using double blank lines before headlines now?

Reply:
yup. it's in the style guide